include/onnxruntime/core/session/onnxruntime_cxx_api.h

```diff
@@ -881,6 +885,39 @@ struct PrepackedWeightsContainer : detail::Base<OrtPrepackedWeightsContainer> {
   PrepackedWeightsContainer();
 };
 
+// 3) Add this implementation block (namespace Ort::detail), e.g. after other *Impl classes:
+
+namespace detail {
+template <typename T>
+struct ExternalInitializerInfoImpl : Base<T> {
```

Comment:
Does this need to be `ConstExternalInitializerInfoImpl` as you noted in this other PR: https://github.com/microsoft/onnxruntime/pull/25455#discussion_r2311491769